### PR TITLE
Enh/doc install paraview

### PR DESCRIPTION
**Pull request by akhoubani**

### CODE REVIEW

File: docs/install_opt.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+   conda install -c conda-forge paraview
+
+- In ArchLinux, the `ParaView package is built with system python`_, one can use::
+   
+   sudo pacman -S paraview; python -c 'import paraview'
+
+.. _paraview package is built with system python: https://github.com/archlinux/svntogit-community/blob/packages/paraview/trunk/PKGBUILD
+
+- The ParaView way of using ``pvpython`` tool is a bit complicated. And the most common hack is to append the virtual environment site-packages path to ``sys.path`` or ``$PYTHONPATH``. This would be the last resort.
```
Comment (ashwinvis) [inline docs/install_opt.rst:16]: Some more clarifications are needed here.

```rst
In many ParaView distributions, the package is not packaged along with the system ``python``. Instead it comes with a
separate ``pvpython`` executable. Installing ``snek5000`` into ``pvpython`` or to use ``snek5000`` in the "Python Shell" 
in ParaView GUI is a bit complicated. On possible hack is to create a virtual environment using the same Python 
version as in ``pvpython`` and append the virtual environment's site-packages path to ``sys.path`` or environment 
variable ``$PYTHONPATH``.
```

File: docs/install_opt.rst [hunk 2]
```diff
@@ -0,0 +1,17 @@
+Installing ParaView module
+==========================
+
+To install Paraview module:
```
[the remark this answers is not in the document]
Comment (ashwinvis) [reply docs/install_opt.rst:4]: You could mention in a note that using Paraview as a reader is a work in progress.

**Review by ashwinvis — CHANGES_REQUESTED** [review]

File: docs/install.rst [hunk 3]
```diff
@@ -53,3 +53,12 @@ interpreter of your choice:
 .. _PyPy: https://www.pypy.org/
 .. _conda: https://docs.conda.io/projects/conda/en/latest/user-guide/index.html
 .. _mamba: https://mamba.readthedocs.io/en/latest/installation.html
+
+
+Installing ParaView module
+==========================
+
+.. toctree::
```
[the remark this answers is not in the document]
Comment (akhoubani) [reply docs/install.rst]: Done!

File: docs/install_opt.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (akhoubani) [reply docs/install_opt.rst]: Done!

File: docs/install_opt.rst (hunk 1 above)
Comment (akhoubani) [reply docs/install_opt.rst]: Done!

**Pull request merged**